airflow/example_dags/example_http_operator.py

```diff
@@ -40,55 +40,56 @@
 
 # t1, t2 and t3 are examples of tasks created by instantiating operators
 t1 = SimpleHttpOperator(
+    http_conn_id='http_for_example_dag',
     task_id='post_op',
-    endpoint='api/v1.0/nodes',
+    endpoint='/http_test',
     data=json.dumps({"priority": 5}),
     headers={"Content-Type": "application/json"},
-    response_check=lambda response: True if len(response.json()) == 0 else False,
+    response_check=lambda response: True if len(response.json()) > 0 else False,
     dag=dag)
 
 t5 = SimpleHttpOperator(
+    http_conn_id='http_for_example_dag',
     task_id='post_op_formenc',
-    endpoint='nodes/url',
+    endpoint='/http_test',
     data="name=Joe",
     headers={"Content-Type": "application/x-www-form-urlencoded"},
     dag=dag)
 
 t2 = SimpleHttpOperator(
+    http_conn_id='http_for_example_dag',
     task_id='get_op',
     method='GET',
-    endpoint='api/v1.0/nodes',
+    endpoint='/http_test',
     data={"param1": "value1", "param2": "value2"},
     headers={},
     dag=dag)
 
-t3 = SimpleHttpOperator(
-    task_id='put_op',
-    method='PUT',
-    endpoint='api/v1.0/nodes',
-    data=json.dumps({"priority": 5}),
-    headers={"Content-Type": "application/json"},
-    dag=dag)
+# t3 = SimpleHttpOperator(
+#     task_id='put_op',
+#     method='PUT',
+#     endpoint='/message/123',
+#     data=json.dumps({"priority": 5}),
+#     headers={"Content-Type": "application/json"},
+#     dag=dag)
 
-t4 = SimpleHttpOperator(
-    task_id='del_op',
-    method='DELETE',
-    endpoint='api/v1.0/nodes',
-    data="some=data",
-    headers={"Content-Type": "application/x-www-form-urlencoded"},
-    dag=dag)
+# t4 = SimpleHttpOperator(
+#     task_id='del_op',
+#     method='DELETE',
+#     endpoint='/message/123',
+#     data="some=data",
+#     headers={"Content-Type": "application/x-www-form-urlencoded"},
+#     dag=dag)
 
 sensor = HttpSensor(
     task_id='http_sensor_check',
-    http_conn_id='http_default',
-    endpoint='',
+    http_conn_id='http_for_example_dag',
+    endpoint='/http_test',
     params={},
-    response_check=lambda response: True if "Google" in response.content else False,
+    response_check=lambda response: True if "true" in response.text else False,
```

Comment:
From https://github.com/apache/incubator-airflow/pull/1753, you can change this to `lambda response: 'true' in response.text`